static/src/javascripts-legacy/projects/commercial/modules/hosted/gallery.js

```diff
@@ -450,32 +450,29 @@ define([
         start();
 
         if (qwery('.js-hosted-gallery-container').length) {
-            loadCssPromise
-            .then(function () {
-                var gallery,
-                    match,
-                    galleryHash = window.location.hash,
-                    res;
-
-                gallery = new HostedGallery();
-                match = /\?index=(\d+)/.exec(document.location.href);
-                if (match) { // index specified so launch gallery at that index
-                    gallery.loadAtIndex(parseInt(match[1], 10));
-                } else {
-                    res = /^#(?:img-)?(\d+)$/.exec(galleryHash);
-                    if (res) {
-                        gallery.loadAtIndex(parseInt(res[1], 10));
+            return loadCssPromise
+                .then(function () {
+                    var gallery,
+                        match,
+                        galleryHash = window.location.hash,
+                        res;
+
+                    gallery = new HostedGallery();
+                    match = /\?index=(\d+)/.exec(document.location.href);
+                    if (match) { // index specified so launch gallery at that index
+                        gallery.loadAtIndex(parseInt(match[1], 10));
+                    } else {
+                        res = /^#(?:img-)?(\d+)$/.exec(galleryHash);
+                        if (res) {
+                            gallery.loadAtIndex(parseInt(res[1], 10));
+                        }
                     }
-                }
-                return gallery;
-            })
-            .then(function (gallery) {
-                stop();
-                mediator.emit('page:hosted:gallery', gallery);
-            });
-        } else {
-            stop();
+                    stop();
+
+                    return gallery;
+                });
         }
+        stop();
```

Comment:
You can now get rid of these if you change [this line](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/bootstraps/commercial.js#L76) to:
```
['cm-hostedGallery', hostedGallery.init],
```